packages/devtools/src/launcher.js

```diff
@@ -58,6 +57,7 @@ async function launchChrome (capabilities) {
 }
 
 function launchFirefox (capabilities) {
+    puppeteerFirefox = require('puppeteer-firefox');
```

Comment:
This has to be defined

```suggestion
    const puppeteerFirefox = require('puppeteer-firefox');
```

Reply:
oops my bad, ok.